packages/splits-sdk/src/utils/numbers.ts

```diff
@@ -15,6 +15,10 @@ export const getBigIntFromPercent = (value: number): bigint => {
   return BigInt(Math.round(Number(PERCENTAGE_SCALE) * value) / 100)
 }
 
+export const getNumberFromPercent = (value: number): number => {
```

Comment:
nit: use this in `getBigIntFromPercent`, right? it's just that part inside of `BigInt(...)`?